src/uwtools/tests/test_rocoto.py

```diff
@@ -129,6 +129,43 @@ def test__add_task_dependency_fail(self, instance, root):
         with raises(UWConfigError):
             instance._add_task_dependency(e=root, config=config)
 
+    @pytest.mark.parametrize(
+        "config", [{"datadep": {"attrs": {"age": "120"}}}, {"timedep": {"attrs": {"offset": "&DEADLINE;"}}}]
+    )
+    def test__add_task_dependency_operand(self, config, instance, root):
+        instance._add_task_dependency_operand(e=root, config=config)
+        element = root[0]
+        for tag, attrs in config.items():
+            assert tag == element.tag
+            for attr, val in attrs.items():
+                assert element.get(attr) == val
```

Comment:
I am having a bit of a problem tracking this error, where I get an error of
```
AssertionError: assert None == {'age': '120'}
AssertionError: assert None == {'offset': '&DEADLINE;'}
```
When using the debugger I haven't been able to trace down what is happening since I am having a hard time accessing the `element` variable. I included a little snippet from the debugger itself below:

```
> /Users/naureenbharwani/Desktop/UFS/workflow-tools/src/uwtools/tests/test_rocoto.py(139)test__add_task_dependency_operand()
(Pdb) p config
{'datadep': {'attrs': {'age': '120'}}}
(Pdb) n
-> for tag, attrs in config.items():
(Pdb) n
-> assert tag == element.tag
(Pdb) p tag
'datadep'
(Pdb) p element.tag
'datadep'
(Pdb) n
-> for attr, val in attrs.items():
(Pdb) n
-> assert element.get(attr) == val
(Pdb) p attr
'attrs'
(Pdb) p val
{'age': '120'}
(Pdb) p element.get(attr)
**None**
(Pdb) p attr
'attrs'
(Pdb) 

```

Reply:
```
(Pdb) ll
132  	    @pytest.mark.parametrize(
133  	        "config", [{"datadep": {"attrs": {"age": "120"}}}, {"timedep": {"attrs": {"offset": "&DEADLINE;"}}}]
134  	    )
135  	    def test__add_task_dependency_operand(self, config, instance, root):
136  	        breakpoint()
137  	        instance._add_task_dependency_operand(e=root, config=config)
138  	        element = root[0]
139  	        for tag, attrs in config.items():
140  ->	            assert tag == element.tag
141  	            for attr, val in attrs.items():
142  	                assert element.get(attr) == val
(Pdb) p tag, attrs
('datadep', {'attrs': {'age': '120'}})
```
I think `attrs` isn't what's needed. Try
```
% git diff -U0
diff --git a/src/uwtools/tests/test_rocoto.py b/src/uwtools/tests/test_rocoto.py
index a73272f..f48e606 100644
--- a/src/uwtools/tests/test_rocoto.py
+++ b/src/uwtools/tests/test_rocoto.py
@@ -140 +140 @@ class Test_RocotoXML:
-            for attr, val in attrs.items():
+            for attr, val in attrs["attrs"].items():
```